test/uk/gov/hmrc/helptosavefrontend/repo/SessionStoreSpec.scala

```diff
@@ -89,28 +89,30 @@ class SessionStoreSpec extends TestSupport with MongoSupport with ScalaFutures w
         result2.value.futureValue should be(Right(()))
 
         val getResult = sessionStore.get(htsSessionReads, hc)
-        getResult.value.futureValue should be(Right(Some(htsSession.copy(ivURL        = Some(ivUrl), ivSuccessURL = Some(ivSuccessUrl)))))
+        getResult.value.futureValue should be(Right(Some(htsSession.copy(ivURL = Some(ivUrl), ivSuccessURL = Some(ivSuccessUrl)))))
 
       }
     }
 
+    //Commenting this test intentionally
+    //Enable this locally if you want to test the expiry of sessions after TTL period is passed
     //time taking unit test due to Eventually
-    "be able to delete entries automatically after TTL period is expired" in new TestApparatus {
-
-      htsSessionGen.sample.foreach { htsSession ⇒
-
-        val hc: HeaderCarrier = HeaderCarrier(sessionId = Some(SessionId(UUID.randomUUID().toString)))
-        val result = sessionStore.store(htsSession)(htsSessionWrites, hc)
-
-        result.value.futureValue should be(Right(()))
-
-        eventually {
-          val getResult = sessionStore.get(htsSessionReads, hc)
-          getResult.value.futureValue should be(Right(None))
-        }
-
-      }
-    }
+    //    "be able to delete entries automatically after TTL period is expired" in new TestApparatus {
```

Comment:
is there a reason we don't want to leave this uncommented out? Is it the time it takes for the unit test?

Reply:
yeah, this test some times runs upto 1 minute - don't want to run the unit tests long so commented out -